jbmc/unit/java-testing-utils/load_java_class.cpp

```diff
@@ -173,3 +179,22 @@ symbol_tablet load_java_class(
   return load_java_class(
     java_class_name, class_path, main, std::move(java_lang), command_line);
 }
+
+std::unique_ptr<goto_modelt> load_goto_model_from_java_class(
+  const std::string &java_class_name,
+  const std::string &class_path,
+  const std::string &main)
+{
+  free_form_cmdlinet command_line;
+  command_line.add_flag("no-lazy-methods");
+  command_line.add_flag("no-refine-strings");
+
+  register_language(new_java_bytecode_language);
+
+  return load_goto_model_from_java_class(
+    java_class_name,
+    class_path,
+    main,
+    get_language_from_mode(ID_java),
```

Comment:
:question: can't we pass the new language directly instead of registering it and then getting it?

Reply:
At the moment you can't, but the whole language registration thing will be removed anyway soonish...

Reply:
I can (in fact elsewhere it is done when providing a custom language):
```C++
std::unique_ptr<languaget> lang = new_java_bytecode_language();
..
main,
std::move(lang),
```

Have done this as reduces dependencies on external code so unless there is a reason this is wrong @peterschrammel ?